src/lib/netlist/plib/ppmf.h

```diff
@@ -8,28 +8,24 @@
 /// \file ppmf.h
```

Comment:
This file seems to have inadvertently had its execute bit set.

Reply:
Strange. Will be fixed.